src/app/shared/components/paragraph-oe-banner/paragraph-oe-banner.component.html

```diff
@@ -1,5 +1,6 @@
 <ecl-banner
     [variant]="oeParagraphsVariant"
+    [eclSize]="fieldOeBannerSize"
     [class.full-width]="fieldOeBannerFullWidth"
```

Comment:
this was fixed in the pr about graphQL, like this we get a class "full-width", the component should render instead `ecl-banner--full-width`, but it's the eUi component adding the class based on the boolean value for the isfullWidth input:
`isFullWidth="fieldOeBannerFullWidth"`

Reply:
Then, i know that this is pr about something else but: there is some missing  ngIf in this file, the description is optional and the cta link as well.
For the cta link in particular there is an accessor "hasCtaLink", i've tried to use that instead of putting ifs but i didn't work, in a way or another, in any case, we need to ensure not to render this markup if there is no request for it.